index.md

```diff
@@ -12,19 +12,20 @@ spend a lot of time waiting for the other to finish, but if they work
 on their own copies and email changes back and forth things will be
 lost, overwritten, or duplicated.
 
-A colleague suggests using [version control](reference.html#version-control) to manage their work.
-Version control is better than mailing files back and forth because:
+A colleague suggests using [version control](reference.html#version-control) to
+manage their work. Version control is better than mailing files back and forth:
+
+*   Nothing that is committed to version control is ever lost.  This means it
```

Comment:
Replace "This means it..." with Version control's revert feature can be used like